src/modules/mc_hover_thrust_estimator/MulticopterHoverThrustEstimator.hpp

```diff
@@ -105,6 +104,8 @@ class MulticopterHoverThrustEstimator : public ModuleBase<MulticopterHoverThrust
 	bool _landed{false};
 	bool _in_air{false};
 
+	bool _was_valid{false};
```

Comment:
This variable name is not very clear. What about `_shoud_invalidate_status`?

Reply:
It's simply a flag storing the last validity. It's not used by the `publishStatus()` and the invalid publish cases so we don't bother continuously publishing empty updates.

Renamed `_was_valid` -> `_valid`.